bin/lint

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 set -e
 
-find src -name 'git-*' -print0 | xargs -0 shellcheck --exclude=SC2155
-find src -name '*.sh' -print0 | xargs -0 shellcheck --exclude=SC2155
-find bin -type f -print0 | xargs -0 shellcheck --exclude=SC2155
+# Exluded rules (see https://github.com/koalaman/shellcheck/wiki/)
+#
+# SC1090: we aready run shellcheck on all our source files
+# SC2155: would bloat the code base
+# SC2162: read -r option not available on Mac OSX 10.10
+
+exclude="SC1090,SC2155,SC2162"
+
+find bin src -type f ! -name 'README.md' -print0 | xargs -0 shellcheck --exclude="$exclude"
```

Comment:
`find bin src -type f ! -iregex '.*\.md' -print0 | ...` to be a tiny bit more robust